search-index/app/load_data.py

```diff
@@ -16,16 +16,115 @@ def get_data_from_navigator_tables(
         pd.DataFrame: one row per document.
     """
     query = """
-        SELECT document_id, source_url, s3_url, document.language_id as document_language_id, document.name AS document_name, action.*, language.language_id, language.language_code, language.name as language_name, \
-        geography.*, source.source_id, source.name as action_source_name, action_type.action_type_id, action_type.type_name as action_type_name, action.name as action_name, action.description as action_description, \
-        geography.country_code as action_country_code, geography.english_shortname as action_geography_english_shortname
-        FROM document
-        INNER JOIN action ON (document.action_id = action.action_id)
-        LEFT JOIN language on (document.language_id = language.language_id)
-        LEFT JOIN geography on (action.geography_id = geography.geography_id)
-        LEFT JOIN source on (action.action_source_id = source.source_id)
-        LEFT JOIN action_type on (action.action_type_id = action_type.action_type_id);
-        """
+      SELECT
```

Comment:
This has everything required by 480 and more (in case we need to update).

Reply:
Just manually checked and seems like languages in the database are broken. Will raise on Slack and Zenhub